code/RealIPProcessor.php

```diff
@@ -1,8 +1,17 @@
 <?php
 
-class AuditLoggerFormatter extends SS_LogErrorFileFormatter
+namespace SilverStripe\Auditor;
+
+class RealIPProcessor
 {
-    protected function getClientIP()
+
+    public function __invoke(array $record)
+    {
+		$record['extra']['real_ip'] = $this->getClientIP();
+		return $record;
+	}
+
+    private function getClientIP()
     {
         $ipaddress = '';
         if (@$_SERVER['HTTP_CLIENT_IP']) {
```

Comment:
See SS_HTTPRequest::getIP(); This implementation follows proper header trust rules, to make IP spoofing harder. :)